README.md

```diff
@@ -189,6 +189,7 @@ Accepted values are:
 -   `all` (default) - Will annotate sections of your code that failed tests or test did not cover
 -   `none` - Turns off annotations
 -   `coverage` - Will annotate those sections of your code that test did not cover
+-   `changed-coverage` - Will only annotate those sections of your code you changed that test did not cover
```

Comment:
Maybe make `changed-coverage` work as default behaviour? I think there is no reason to create a new value, the value `coverage` could just annotate changed sections of code.

Reply:
That would be my use case but I wasn't sure if that was what you intended so I added an option. If you say this should be default behaviour on pull requests, I'm happy to